services/jobrunner/defaults.env

```diff
@@ -29,6 +29,8 @@ WORKDIR=/home/opensafely/jobrunner/workdir
 AIRLOCK_WORK_DIR=/workdir/
 AIRLOCK_WORKSPACE_DIR=/workspaces/
 AIRLOCK_REQUEST_DIR=/requests/
+DJANGO_DEBUG=False
+DJANGO_ALLOWED_HOSTS=*
```

Comment:
it doesn't feel great setting this here without namespacing them, but :shrug:

Reply:
also, we could probably set allowed_hosts tighter if you like?

Reply:
I think you could set both these in the `env:` section of docker-compose.yaml?

Onlyy need to be in config files if a) we expect to change them and/or b) they are shared between apps.

Reply:
In fact, the AIRLOCK_*_DIR paths could all be hardcoded on in the docker.yaml, no? I can't see why we'd ever want to change them?

Reply:
> Can it be $RELEASE_HOST?

it can be $HOSTNAME? 

https://github.com/opensafely-core/backend-server/blob/main/tests/check-release-hatch.sh#L7-L8

Reply:
> the AIRLOCK_*_DIR paths could all be hardcoded on in the docker.yaml, no?

yeah that's a good point - I just cribbed what release-hatch did, but you're right that they don't need to be there, we can move them.